kroma-chain-ops/genesis/layer_one.go

```diff
@@ -140,7 +142,68 @@ func PostProcessL1DeveloperGenesis(stateDB *state.MemoryStateDB, deployments *L1
 	val := stateDB.GetState(deployments.ProxyAdmin, common.BigToHash(common.Big0))
 
 	stateDB.SetState(deployments.SystemConfigProxy, slot, val)
-	log.Info("Post process update", "address", deployments.SystemConfigProxy, "slot", slot.Hex(), "value", val.Hex())
+	log.Info("Post process update", "name", "SystemConfig", "address", deployments.SystemConfigProxy, "slot", slot.Hex(), "value", val.Hex())
+
+	// Change the key of _quorumNumeratorHistory in UpgradeGovernor to 1 which means that quorumNumerator has been set at L1 block number 1 for guardian test
+	if !stateDB.Exist(deployments.UpgradeGovernorProxy) {
+		return fmt.Errorf("upgardeGovernor proxy doesn't exist at %s", deployments.UpgradeGovernorProxy)
+	}
+
+	layout, err = bindings.GetStorageLayout("UpgradeGovernor")
+	if err != nil {
+		return errors.New("failed to get storage layout for UpgradeGovernor")
+	}
+
+	entry, err = layout.GetStorageLayoutEntry("_quorumNumeratorHistory")
+	if err != nil {
+		return errors.New("failed to get storage layout entry for UpgradeGovernor._quorumNumeratorHistory")
+	}
+	slot = common.BigToHash(big.NewInt(int64(entry.Slot)))
+	slot = crypto.Keccak256Hash(slot.Bytes())
+
+	beforeVal := stateDB.GetState(deployments.UpgradeGovernorProxy, slot)
+	checkpointVal := make([]byte, 28)
+	copy(checkpointVal, beforeVal[:28])
+	checkpointKey := [4]byte{}
+	checkpointKey[3] = 0x01
```

Comment:
Code duplicates lines 197-200.

Reply:
Yes, but I thought it could be different in future. So I remained it separately.